modules/mcuboot/boot/zephyr/Kconfig

```diff
@@ -25,13 +25,14 @@ source "${ZEPHYR_BASE}/../nrf/subsys/partition_manager/Kconfig.template.partitio
 endif
 
 config PM_PARTITION_SIZE_MCUBOOT
-	hex
+	hex "Flash space allocated for the MCUboot partition"
 	default 0x3e00
 	depends on (BOOT_NRF_EXTERNAL_CRYPTO && BOOT_USE_MIN_PARTITION_SIZE)
 
 config PM_PARTITION_SIZE_MCUBOOT
```

Comment:
I have no idea what is going on here but there should only be one of these symbols defined not 2